src/buildblock/recon_array_functions.cxx

```diff
@@ -346,54 +361,79 @@ void divide_array(DiscretisedDensity<3,float>& numerator, const DiscretisedDensi
 
 //  MJ 03/01/2000 for loglikelihood computation
 // KT 21/05/2001 make sure it returns same result as divide_and_truncate above
+// RT 17/02/2021 explicitly make sure that it returns the same value as (possion_)divide_and_truncate above by using
+// the same functionality
 void accumulate_loglikelihood(Viewgram<float>& projection_data, 
 			 const Viewgram<float>& estimated_projections,
 			 const int rim_truncation_sino,
-			 double* accum)
+			 double* log_likelihood_ptr,
+			 const bool use_KL_divergence)
 {
-  
+#if 1
+  int count = 0;
+  int count2 = 0;
+      poisson_divide_and_truncate(projection_data,
+                                  estimated_projections,
+                                  rim_truncation_sino,
+                                  count, count2,
+                                  log_likelihood_ptr, use_KL_divergence, false);
+
```

Comment:
`accumulate_loglikelihood` now uses the `poisson_divide_and_truncate` method with a `log_likelihood_ptr` and `do_division=false`